scripts/purge-old-deployments

```diff
@@ -14,27 +14,37 @@ def parse_args() -> argparse.Namespace:
     parser = argparse.ArgumentParser(
         description="This script can be used for cleaning old unused deployments.",
         epilog="Orphaned/unused caches older than threshold days will be automatically "
-        "examined and removed.")
+        "examined and removed.",
+    )
     parser.add_argument(
-        "--threshold", dest="threshold_days", type=int, default=14,
-        metavar="<days>", help="Deployments older than "
-        "threshold days will be deleted. (defaults to 14)")
+        "--threshold",
+        dest="threshold_days",
+        type=int,
+        default=14,
+        metavar="<days>",
+        help="Deployments older than " "threshold days will be deleted. (defaults to 14)",
+    )
     parser.add_argument(
-        "--dry-run", action="store_true",
+        "--dry-run",
+        action="store_true",
         help="If specified then script will only print the deployments and "
-        "caches that it will delete/keep back. It will not delete anything.")
+        "caches that it will delete/keep back. It will not delete anything.",
+    )
     parser.add_argument(
-        "--verbose", action="store_true",
-        help="If specified then script will print a detailed report "
-        "of what is going on.")
+        "--verbose",
+        action="store_true",
+        help="If specified then script will print a detailed report " "of what is going on.",
```

Comment:
Can you do a bit of a pre-fix for this category of string-wrapping issue (note the useless `" "` in the middle)?

Reply:
Sure (I think I pre-fixed a bunch of these earlier but I guess there are new ones).

Reply:
Done, and found some bugs along the way: #17282.